test/unit/models/task.test.js

```diff
@@ -0,0 +1,141 @@
+const chai = require('chai')
+const { expect } = chai
+const tasks = require('../../../models/tasks')
+const sinon = require('sinon')
+const addUser = require('../../utils/addUser')
+const userData = require('../../fixtures/user/user')()
+const appOwner = userData[3]
+const otherUser = userData[0]
+
+describe('tasks', function () {
+  before(async function () {
+    await addUser(appOwner)
+    await addUser(otherUser)
+  })
+
+  afterEach(function () {
+    sinon.restore()
+  })
+
+  describe('fetchUserTasks', function () {
+    describe('When user exists and status is not passed', function () {
+      it('Should return blank array if no task are assigned to user', async function () {
+        const allTasks = await tasks.fetchUserTasks(appOwner.username)
+        expect(allTasks).to.eql([])
+      })
+
+      it('Should return group tasks in array', async function () {
+        const taskData = [
+          {
+            title: 'Title of the task 1',
+            type: 'group',
+            endsOn: '1621814400',
+            startedOn: '1621728000',
+            status: 'completed',
+            participants: [
+              appOwner.username
+            ]
+          }]
+
+        await tasks.updateTask(taskData[0])
+        const allTasks = await tasks.fetchUserTasks(appOwner.username)
+        const task1 = allTasks[0]
+        if (task1.type === 'group') {
```

Comment:
we have constants for tasks in constants/task.js -> TASK_STATUS and TASK_TYPE
Better we use them for the task type checks and task status checks